config/reports.json

```diff
@@ -482,7 +482,7 @@
       "name": "Cumulative Layout Shift",
       "type": "",
       "singular": "",
-      "description": "The layout shift scores for every unexpected layout shift that occurs during the entire lifespan of the page. See the [Cumulative Layout Shift (CLS)](https://web.dev/cls/) for more info.",
+      "description": "The layout shift scores for every unexpected layout shift that occurs during the entire lifespan of the page. See the [Cumulative Layout Shift (CLS)](https://web.dev/articles/cls/) for more info.",
```

Comment:
Can we remove final slash from all of these while we're at it?

```suggestion
      "description": "The layout shift scores for every unexpected layout shift that occurs during the entire lifespan of the page. See the [Cumulative Layout Shift (CLS)](https://web.dev/articles/cls) for more info.",
```

Reply:
What exactly does this optimize? 😅

Reply:
Google Analytics, so it doesn't track it as two different pages, for example:

<img width="259" alt="image" src="https://github.com/HTTPArchive/httparchive.org/assets/10931297/528eef83-8e63-4ccb-aa41-c79dc6a18ed2">

We probably should also update it to strip off the final `/`.

Reply:
Ok SG